app/src/test/java/io/gnosis/safe/ui/safe/terms/TermsCheckerTest.kt

```diff
@@ -0,0 +1,55 @@
+package io.gnosis.safe.ui.safe.terms
+
+import android.app.Application
+import io.gnosis.safe.ui.safe.terms.TermsChecker.Companion.TERMS_AGREED
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.spyk
+import junit.framework.Assert.assertFalse
+import junit.framework.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import pm.gnosis.svalinn.common.PreferencesManager
+import pm.gnosis.tests.utils.TestPreferences
+
+class TermsCheckerTest {
+    private lateinit var preferences: TestPreferences
+    private lateinit var preferencesManager: PreferencesManager
```

Comment:
`TestPreferences` from svalinn test utils. Check the `SafeRepositoryTest.kt` for reference, if you need.